jib-core/src/test/java/com/google/cloud/tools/jib/registry/ManifestPullerTest.java

```diff
@@ -89,6 +90,24 @@ public void testHandleResponse_v22()
     Assert.assertThat(manifestTemplate, CoreMatchers.instanceOf(V22ManifestTemplate.class));
   }
 
+  @Test
+  public void testHandleResponse_v22ManifestList()
+      throws URISyntaxException, IOException, UnknownManifestFormatException {
+    Path v22ManifestListFile =
+        Paths.get(Resources.getResource("core/json/v22manifest_list.json").toURI());
+    InputStream v22ManifestList = new ByteArrayInputStream(Files.readAllBytes(v22ManifestListFile));
+
+    Mockito.when(mockResponse.getBody()).thenReturn(v22ManifestList);
+    ManifestTemplate manifestTemplate =
+        new ManifestPuller<>(
+                fakeRegistryEndpointRequestProperties,
+                "test-image-tag",
+                V22ManifestListTemplate.class)
+            .handleResponse(mockResponse);
+
+    Assert.assertThat(manifestTemplate, CoreMatchers.instanceOf(V22ManifestListTemplate.class));
```

Comment:
Maybe it's good to test one or two values, since I think there is a possibility this test may pass even if `v22manifest_list.json` is not a manifest list, knowing unknown fields will be ignored.

Reply:
I think the media type should catch that, but I can play around with this test some more.

Reply:
If a manifest class given is not `ManifestTemplate.class`, the `handleResponse()` code just boils down to

```java
      return JsonTemplateMapper.readJson(jsonString, manifestTemplateClass);
```

so I think it's possible that this test will succeed even if the input is a manifest (not a list), which also has `schemaVerison` (and `mediaType`). I think it's worth checking the existence of the `manifests` field.

Reply:
Can we check the `manifests` field?